component/identity/src/test/java/org/exoplatform/services/organization/TestLDAPOrganization.java

```diff
@@ -136,6 +133,37 @@ public void testFindUser() throws Exception {
     assertNull(test2);
     end();
   }
+  public void testEnabledUserDetection() throws Exception{
```

Comment:
```suggestion

  public void testEnabledUserDetection() throws Exception{
```
Please format your code. Please check http://developer.exoplatform.org/#id-intellij-settings